Ago-Dic-2020/chavez-muñoz-juan-arnoldo/Practica-2/Practica-2.py

```diff
@@ -0,0 +1,41 @@
+class Carro(object):
+    def __init__(self, name_car, año, kilometraje):
+        self.name_car = name_car
+        self.año = año
+        self.kilometraje = kilometraje
+
+    #Getters and Setters
+    def set_name_car(self, name_car):
+        self.name_car = name_car
+
+    def set_año(self, año):
+        self.año = año
+
+    def set_kilometraje(self, kilometraje):
+        self.kilometraje = kilometraje
+
+    def get_name_car(self):
+        return self.name_car
+
+    def get_año(self):
+        return self.año
+
+    def get_kilometraje(self):
+        return self.kilometraje
+
+    def CarIsFlying (self):
```

Comment:
Funciones y variables al estilo `snake_case` por recomendaciones del estándar oficial: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names